src/app/shared/filter/filter.component.scss

```diff
@@ -0,0 +1,51 @@
+.filter-section{
+    padding: 10px;
+    overflow-y: auto;
+    max-height: 100vh;
+}
+
+.filter-header{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    font-size: 20px;
+    font-weight: 700;
+    margin-top: 8px;
+
+    mat-icon{
+        cursor: pointer;
+    }
+}
+
+.clear{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    font-size: 16px;
+    color: var(--color-primary);
+    cursor: pointer;
+    margin: 16px 0px;
+}
+
+.filter-name{
+    font-size: 16px;
+    font-weight: bold;
+}
+
+.filter-options{
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 16px;
+}
+
+.apply-button{
+    position: fixed;
+    bottom: 0;
+    left: 50px;
+    width: 200px;
+    background-color: white;
```

Comment:
should white color for the button?

Reply:
this was added because the other texts was coming under this when scrolling and it was transparent
This was one of the UI issue, will note this and make a fix and push it later with the other UI fixes